opensrp-app/src/main/java/org/smartregister/ssl/OpensrpSSLHelper.java

```diff
@@ -61,4 +64,27 @@ public void verify(String host, String[] cns, String[] subjectAlts) throws
         }
     }
 
+    public javax.net.ssl.SSLSocketFactory getSSLSocketFactory() {
```

Comment:
This method could benefit from constants like `"phone red pen"`, `"X509"` and `"TLS"` being made constants?